api/src/main/java/com/vexsoftware/votifier/model/Vote.java

```diff
@@ -78,7 +78,7 @@ public Vote(Vote vote) {
                 vote.getAdditionalData() == null ? null : vote.getAdditionalData().clone());
     }
 
-    private static String getTimestamp(JsonElement object) {
+    public static String getTimestamp(JsonElement object) {
```

Comment:
Why?

Reply:
You are accessing stuff in you Unit Tests via reflection in another module. In Java 16 illegal Reflection is not possible anymore. I opened the java module there. It could be accessible the old way but I don't see a reason to keep something like this utility method private. The cleaner way would be to move your test cases that validates the functionality of that method to the right module.

Reply:
Please fix the issue without depending on a newer JDK version.